File: Algoritmos Traduzidos/Ruby/Algoritmos_Tiago/Alg_55.rb
#Criar um algoritmo que calcule e imprima a área de um losango.
puts "Introduza a diagonal menor:"
dmenor = gets
puts "Introduza a diagonal maior:"
dmaior = gets
area = (dmenor.to_f * dmaior.to_f) / 2
print "A area e: #{area}"

File: Algoritmos Traduzidos/Visual Basic Console/Tiago/alg_55.vb
Module alg_55
    Sub main()
        Dim dmenor As Double
        dmenor = 0.0
        Dim dmaior As Double
        dmaior = 0.0
        Console.WriteLine("introduza a diagonal menor")
        dmenor = Console.ReadLine()
        Console.WriteLine("introduza a diagonal maior")
        dmaior = Console.ReadLine()
        Dim area As Long
        area = (dmenor * dmaior) / 2
        Console.WriteLine("A area é:" + area.ToString)
        Console.ReadLine()
    End Sub
End Module
